### DISCUSSION TITLE
How to edit the items/entry command in rofi

### DISCUSSION THREAD

**Discussion by YerongLi** [davatorium/rofi#2011] [Q&A] [answered]
How can we edit the linking of an entry appeared in the search bar?

For example, when I search **slack**, an entry `slack` is mistakenly linked to `/usr/sbin/slack` while it should be linked to `/usr/bin/slack` 
```
run: slack (Search)
slack
slack - diff
```

**Comment by DaveDavenport** [comment]
If you use 'run' mode, it follows your PATH environment.

**Reply by YerongLi** [reply, marked as answer]
Not really, whenever I put **bin** before **sbin** `usr/bin:/usr/sbin`, whether it is in the `.profile` for `sudoer`. the command in `usr/sbin` folder still overrides the commands in `usr/bin`.

Even if I can see in `echo $PATH` usr/bin prioritizes over the over `usr/bin:/usr/sbin`

**Reply by DaveDavenport** [reply]
rofi executes the selected command without passing a full path and tells the underlying glib call to find the executable in path. 
It inherits the environment from the program that launches rofi (f.e. your keybinding tool).

https://github.com/davatorium/rofi/blob/next/source/helper.c#L1010
